mandalab_visual/mandalab_visual.py

```diff
@@ -0,0 +1,77 @@
+import datetime
+from pyvis.network import Network
+from mandalab_visual.consts import *
+
+DEFAULT_GRAPH_NAME_FORMAT = "{graph_base}_{name}_{timestamp}.html"
+
+
+def guess_if_running_in_notebook():
+    try:
+        from IPython import get_ipython
+        if get_ipython() is None or 'IPKernelApp' not in get_ipython().config:
+            return False
+    except ImportError:
+        return False
+    return True
+
+
+def generate_graph(machine, notebook=None, show=True, output_html_folder="graphs", output_html_name=None):
+    """
+    This function receives a machine and generate a network graph representing the network that the machine is in.
+    :param machine: The mandalab machine to start the graph with.
+    :param notebook: If None, will try to guess it.
+    :param show: Show the output, or just save it to the output_html_folder. The html file will be generated any way.
```

Comment:
not sure if i understand but maybe a more proper name should be `show` -> `verbose` ?